rust/sdk-processor/src/config/processor_config.rs

```diff
@@ -103,10 +96,44 @@ impl ProcessorConfig {
         }
     }
 
+    /// Get the set of table names to process for the given processor.
+    pub fn table_names(processor: &ProcessorName) -> HashSet<String> {
+        match processor {
+            ProcessorName::ParquetDefaultProcessor => HashSet::from([
+                "move_resources".to_string(),
```

Comment:
sorry i meant that in all places where we have these raw strings, we should use the TABLE_NAME constant defined in each parquet model: 
```
impl NamedTable for MoveResource {
    const TABLE_NAME: &'static str = "move_resources";
}
```